src/alembic/versions/005_775b2cf7ab94_add_token_id_multi_ride_id_to_use_transaction.py

```diff
@@ -0,0 +1,40 @@
+"""add multi_ride_id, token_id columns to use_transaction
+
+Revision ID: 775b2cf7ab94
+Revises: 214f62b149e9
+Create Date: 2024-07-30 15:32:28.457690
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "775b2cf7ab94"
+down_revision: Union[str, None] = "214f62b149e9"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.add_column(
+        "use_transaction_location", sa.Column("multi_ride_id", sa.String())
+    )
+    op.add_column(
+        "use_transaction_location", sa.Column("token_id", sa.String())
+    )
+    op.add_column(
+        "use_transaction_longitudinal", sa.Column("multi_ride_id", sa.String())
+    )
+    op.add_column(
+        "use_transaction_longitudinal", sa.Column("token_id", sa.String())
+    )
```

Comment:
These `add_column` calls have a `sa.String()` type, but the schemas file says they should be `sa.BigInteger`.